pages/index.js

```diff
@@ -1,3 +1,28 @@
+import styled from "styled-components";
+import SpotCard from "../components/SpotCard";
+import spots from "../helpers/mockSpots";
+
 export default function Home() {
-  return <h1>Hello Flamingos! 🦩</h1>;
+  return (
+    <>
+      <Headline>ParentHood Spots</Headline>
+
+      <SpotList>
+        {spots
+          ?.sort((a, b) => a.name.localeCompare(b.name))
+          ?.map((spot) => (
+            <SpotCard spot={spot} key={spot.name}></SpotCard>
+          ))}
+      </SpotList>
+    </>
```

Comment:
For **semantic reasons** I recommend to use a `<main>` element instead of the React fragments `<></>` 
👉 **Another advantage is**: the styles that you previously applied to the entire `<body>` can now be applied to the `<main>` element -- by using  a `sytled component`
```suggestion
    <main>
      <Headline>ParentHood Spots</Headline>

      <SpotList>
        {spots
          ?.sort((a, b) => a.name.localeCompare(b.name))
          ?.map((spot) => (
            <SpotCard spot={spot} key={spot.name}></SpotCard>
          ))}
      </SpotList>
    </main>
```